template.distillery/Makefile.mobile

```diff
@@ -146,20 +146,36 @@ endif
 ##----------------------------------------------------------------------
 ## Rules for static files.
 
+# Function to list files in a given directory $(1), and change prefix directory
+# to the target one $(2)
+# Example: $(call static_targets,$(SOURCE_DIR),$(TARGET_DIR))
+static_targets = $(shell find $(1) -type f -printf "$(2)/%P\n" | grep -v .well-known | grep  -v "/fonts/.*\\([.]svg\\|[.]eot\\|webfont[.]ttf\\)$$")
+
 # Cordova static files which needs to be copied to the Cordova project
 # directory.
 CORDOVA_STATIC_FILES := \
-    $(CORDOVAPATH)/.chcpignore \
-    $(patsubst $(MOBILESTATICPATH)/www/%, \
-               $(CORDOVAPATH)/www/%, \
-               $(shell find $(MOBILESTATICPATH)/www -type f) \
-     )
-
-# FIXME: duplicate rule warning for 'cordova/www/css/os_test.css'
-LOCAL_STATIC_FILES = $(patsubst \
-    $(LOCAL_STATIC)/%, \
-    $(CORDOVAPATH)/www/%, \
-    $(shell find $(LOCAL_STATIC) -type f))
+	$(CORDOVAPATH)/.chcpignore \
+	$(call static_targets,$(MOBILESTATICPATH)/www,$(CORDOVAPATH)/www)
+
+LOCAL_STATIC_FILES := $(call static_targets,$(LOCAL_STATIC)/css,$(CORDOVAPATH)/www/css) $(call static_targets,$(LOCAL_STATIC)/fonts,$(CORDOVAPATH)/www/fonts)
+
+$(CORDOVAPATH)/www/chcp.manifest: $(APPJS) $(APPCSS) \
```

Comment:
The rules are separated depending on what they do. I think around line 250, rules about CHCP are already present.